### ISSUE TITLE
实际场景

### ISSUE THREAD

**Issue by huzhanxiaoxue** [zmliao/Fisheye-GS#6]
Have you actually run tests with a fisheye camera? The paper does not provide any actual test results conducted with a fisheye camera.

**Comment by zmliao** [comment]
Yes I have proviced the result. But the distortion is not apparent yet. Notice the difference on the edge.
